File: V.Vennela/Basics of Programming/Patterns/Pattern13.java
import java.util.*;
public class Pattern13{
    public static void main(String[] args){
        Scanner scn = new Scanner(System.in);

        //write your code here
        int n = scn.nextInt();
        for(int i=1;i<=n;i++)
        {
            int c = 1;
            for(int j=1;j<=i;j++)
            {
                 System.out.print(c+"\t");
                 c=c*(i-j)/j;
            }
                System.out.println();
        }

    }
}

File: Kritika Tripathi/Basics Of Programming/Patterns/Pattern13.cpp
#include <iostream>
using namespace std;

int factorial(int n){
    if(n==1){
        return 1;
    }
    if(n==0){
        return 1;
    }
    return n*factorial(n-1);
}

int main(){
    int n;
    cin >> n;

    int ans;
    
    //write your code here
    for(int i=0; i<=n; i++){
        for(int j=0; j<=i; j++){
            if(j==0 || j==i){
                cout<<"1\t";
            }
            else{
                ans = factorial(i)/(factorial(j)*factorial(i-j));
                cout<<ans<<"\t";
            }
        }
        cout<<endl;
    }
}
